README.md

```diff
@@ -1,75 +1,40 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+## Project Name
+Hello Microverce.
 
+## Description of project.
+School project.
 
 ## Built With
+Html. Css.
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
+## Prerequisites
+Vscode.
 
-### Usage
+## Website link: 
 
-### Run tests
 
-### Deployment
+## Setup
+Clone this repo with:
+https://github.com/eerapheal/Hello-Microverce.git
 
+## Install
+npm install
 
-
-## Authors
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+## Author
+Ekpenisi Erue Raphael. GitHub: (https://github.com/eerapheal) Twitter: (https://twitter.com/ekpenisiraphael) LinkedIn: (https://www.linkedin.com/in/ekpenisi-e-raphael-9678a221a/)
 
 ## 🤝 Contributing
-
 Contributions, issues, and feature requests are welcome!
-
 Feel free to check the [issues page](../../issues/).
 
 ## Show your support
-
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
+My Team.
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
-## 📝 License
-
-This project is [MIT](./LICENSE) licensed.
+License
+This project is Ralph licensed.
```

Comment:
- [ ] As suggested by the previous reviewer, I kindly suggest that you customize your MIT license by putting your name on it, and editing the other placeholders.
![image](https://user-images.githubusercontent.com/52934128/185043262-efddaab5-4c70-4bb3-822b-c09abe57ded0.png)